lib/document/Document.js

```diff
@@ -140,16 +143,16 @@ class Document {
   /**
    * Set entropy
    *
-   * @param {string} entropy
+   * @param {Buffer} entropy
```

Comment:
All EncodedBuffer setters accept strings too

Reply:
I think it's better to use strong typings to avoid a mess